src/modules/Progress/Progress.js

```diff
@@ -1,110 +1,180 @@
-import $ from 'jquery'
-import classNames from 'classnames'
-import React, { Component, PropTypes } from 'react'
+import _ from 'lodash'
```

Comment:
This file is a complete rewrite and best [viewed directly](https://github.com/TechnologyAdvice/stardust/blob/9930ae7991c471e0f651bb3b919b1bf615bf2c37/src/modules/Progress/Progress.js).